kmm/kmm-post-test-fixtures/src/commonMain/kotlin/com/gchristov/newsfeed/CalculateReadingTimeTest.kt

```diff
@@ -0,0 +1,15 @@
+package com.gchristov.newsfeed
+
+import com.gchristov.newsfeed.kmmpostdata.util.ReadingTimeCalculator
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class CalculateReadingTimeTest {
```

Comment:
This still looks like it's in the incorrect place? Should be under `commonTest`

Reply:
good catch, I thought I deleted it. It is superseded by the unit test and potential new UI ones.

Reply:
Ah of course, that makes sense 👍

Reply:
this is now removed